src/metabase/pulse/render/body.clj

```diff
@@ -527,14 +527,16 @@
         "line"))
 
 (defn- join-series
-  [names colors types row-seqs y-axis-positions]
-  (vec (for [[card-name card-color card-type rows y-axis-position]
-             (map vector names colors types row-seqs y-axis-positions)]
-         {:name          card-name
-          :color         card-color
-          :type          card-type
-          :data          rows
-          :yAxisPosition y-axis-position})))
+  [names types row-seqs y-axis-positions y-col]
+  (for [[idx [card-name card-type rows y-axis-position]]
+        (map-indexed vector (map vector names types row-seqs y-axis-positions))]
+    {:name          card-name
+     :type          card-type
+     :data          rows
+     :yAxisPosition y-axis-position
+     :seriesKey     (cond 
+                      (= idx 0) (:name y-col)
+                      :else card-name)}))
```

Comment:
Use the same logic in https://github.com/metabase/metabase/blob/d9a5708306f464e5d512c5d2b7cd18677a1f7667/frontend/src/metabase/visualizations/components/LineAreaBarChart.jsx#L532-L534

This is needed since `y-col` e.g. `"sum"` isn't exposed to FE static viz.